### PR TITLE
Cult Leader Nerf

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Credit goes to Robustin for the summon-related change, just resurrecting this since their PR went stale a while ago. #41082

## Why It's Good For The Game
capture
basically, prevents the above meta of 4x barrier spam and pull every cultist into your lap for an insta-win summon with little counter to it aside from nullrod and bombs. **You'll need to summon your cultists outside of the ritual-area** and they'll need to walk onto it. (Wow, teamwork!)

capture2

## Changelog
:cl: SuicidalPickles
balance: Cult leaders can no longer mass-summon their cultists into the ritual area.
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
I thought Flag robes were still balanced, considering they give 3X damage on any attack you take if you wear them. That was at least what I was told.

**Comment by Suicidalpickles** [comment]
> 
> 
> I thought Flag robes were still balanced, considering they give 3X damage on any attack you take if you wear them. That was at least what I was told.

yeah they still give you extra received damage, but imo they shouldn't be [zoom onto somebody's screen and click them within a second without them being able to react.] Either way they're still fast though compared to folk running around at 1.5 delay.

**Comment by 4dplanner** [comment]
These should be two separate prs imo

**Comment by Suicidalpickles** [comment]
> 
> 
> These should be two separate prs imo

yeah gave it a bit of afterthought, gonna tick flagrobes on it back to -1

**Comment by imsxz** [comment]
as someone who gets cult too often and does this every single time, good PR. 1 step closer to making cult a team mode instead of "1 person is able to do everything"

**Comment by wesoda25** [comment]
fuck cult

**Comment by Tlaltecuhtli** [comment]
is the mass summon even needed to cult?

File: code/modules/antagonists/cult/cult_comms.dm [hunk 1]
```diff
@@ -151,6 +151,11 @@
 	var/datum/antagonist/cult/antag = owner.mind.has_antag_datum(/datum/antagonist/cult,TRUE)
 	if(!antag)
 		return
+	var/datum/objective/eldergod/summon_objective = locate() in antag.cult_team.objectives
+	var/area/place = get_area(owner)
+	if(place in summon_objective.summon_spots)
```
Comment (Cyberboss) [inline code/modules/antagonists/cult/cult_comms.dm]:
```suggestion
	if(place in summon_objective?.summon_spots)
```

You shouldn't assume the cult has a summon objective

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/cult/cult_comms.dm [hunk 2]
```diff
@@ -151,6 +151,11 @@
 	var/datum/antagonist/cult/antag = owner.mind.has_antag_datum(/datum/antagonist/cult,TRUE)
 	if(!antag)
 		return
+	var/datum/objective/eldergod/summon_objective = locate() in antag.cult_team.objectives
+	var/area/place = get_area(owner)
+	if(place in summon_objective?.summon_spots)
```
Comment (Cyberboss) [inline code/modules/antagonists/cult/cult_comms.dm]: Wait, why are you comparing this against every single summonable area? Just get the turfs used by the ritual and filter those out

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/cult/cult_comms.dm (hunk 2 above)
Comment (Suicidalpickles) [reply code/modules/antagonists/cult/cult_comms.dm]: Wouldn't that allow the leader to loop around this by summoning their cultists just a few moments right before the ritual rune is finished on those turfs though? Or am I misunderstanding? Also I'm not sure how to go about with that, just was pasting this from a dead PR.

Comment (Cyberboss) [reply code/modules/antagonists/cult/cult_comms.dm]: areas contain hundreds of turns, this would ban say, all of medbay, or all of science

Comment (Cyberboss) [reply code/modules/antagonists/cult/cult_comms.dm]: Any area where you could summon

Comment (ExcessiveUseOfCobblestone) [reply code/modules/antagonists/cult/cult_comms.dm]: What would stop you from just starting the ritual prior to starting the narsie rune

**Pull request closed without merging**